src/examples/colors/util.py

```diff
@@ -1,12 +1,16 @@
 import argparse
 
-GENERATE_IB_BOUND=False #True to generate the IB bound for the specified parameters
-USE_RKK = False #Whether to use the RKK metric for complexity
-USE_NOGA_ARRAYS = False #True to use the Zaslavsky data for the IB bound, false otherwise
-GENERATE_LANG_COLOR_INFO=True #True to generate color information for each language
-GENERATE_ADDITIONAL_COLOR_CHIPS = False #If true, will expand color terms based off color distance for both natural and artificial languages
-USE_ONE_LANGUAGE = False #If true, just uses the first language in the list of languages
-COLOR_CHIP_THRESHOLD = 5 #The minimum number of color chips a language must have to be included in the analysis
+GENERATE_IB_BOUND = False  # True to generate the IB bound for the specified parameters
+USE_RKK = False  # Whether to use the RKK metric for complexity
+USE_NOGA_ARRAYS = (
+    False  # True to use the Zaslavsky data for the IB bound, false otherwise
+)
+GENERATE_LANG_COLOR_INFO = True  # True to generate color information for each language
+GENERATE_ADDITIONAL_COLOR_CHIPS = False  # If true, will expand color terms based off color distance for both natural and artificial languages
+USE_ONE_LANGUAGE = (
+    False  # If true, just uses the first language in the list of languages
+)
+COLOR_CHIP_THRESHOLD = 5  # The minimum number of color chips a language must have to be included in the analysis
```

Comment:
Why are these being used as constants here versus args to be parsed from command line?